spec/rental_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../book'
+require_relative '../person'
+require_relative '../rental'
+
+describe Rental do
+  before :each do
+    @person = Person.new(1, 'Person1')
+    @book = Book.new('Title1', 'Author1')
+    @rental = Rental.new('2000/01/01', @person, @book)
+  end
+
+  context 'rental test' do
+    it 'book and person must contain added values' do
+      expect(@rental.date).to eq '2000/01/01'
+      expect(@rental.person.rentals.length).to eq(0)
+      expect(@rental.book.rentals.length).to eq(0)
+    end
+  end
```

Comment:
- [ ] The expected length of the `@Book` and `@Person` `rentals` array property should be `1` since you are creating a new `Rental` instance. Remember in the milestone for defining associations, the rental instance pushes itself to the array of the `Person` and `Book` instances. Kindly fix the logic in the `initialize` method of the `Rental` class by removing the `if` statement when pushing the `rental` instance itself into the `rentals` array for the `Book` and `Person` instances. 👍 💻